lib/composer.js

```diff
@@ -250,6 +250,11 @@ export default class Composer {
     return !filePath || filePath.search(/\.(tex|lhs)$/) > 0
   }
 
+  isKnitrFile (filePath) {
+    // scope: text.tex.latex.knitr
+    return !filePath || filePath.search(/\.(Rnw|Snw|rnw|snw)$/) > 0
+  }
+
```

Comment:
Remove this function. I realize you were told to add this, but I don't see why it's necessary.